content/v2.0/monitor-alert/templates/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Monitor with templates
```

Comment:
Again, I don't think this section should be pigeon-holed to using templates. The bulk certainly will, but I think we should keep it more general. Suggestions above. Open to others.